nerdlandbot/eventhandlers/oncommanderror.py

```diff
@@ -18,18 +21,22 @@ async def on_command_error(self, ctx: commands.Context, error: Any):
         :param ctx: The current context. (discord.ext.commands.Context)
         :param error: The current error. (Any)
         """
-        
+        #Notify user when using a command in DM that is not meant for DM
+        if isinstance(error, commands.NoPrivateMessage):
+            msg = translate("err_command_no_DM", await culture_id(int(NERDLAND_SERVER_ID)))
+            return await ctx.send(msg)
         # Notify user for MissingRequiredArgument errors
-        if isinstance(error, commands.MissingRequiredArgument):
+        elif isinstance(error, commands.MissingRequiredArgument):
             command_name = ctx.message.content.split(" ")[0]
-            msg = translate("err_missing_parameter", await culture(ctx)).format(command_name, error.param.name)
+            msg = translate("err_missing_parameter", await culture_ctx(ctx)).format(command_name, error.param.name)
             return await ctx.send(msg)
+
         else:
             # Log the warning
             log_warn(error)
 
         # Notify user with general error
-        msg = translate("err_unrecognized_command", await culture(ctx))
+        msg = translate("err_unrecognized_command", await ctx(ctx))
```

Comment:
I believe this is a typo. It gives this error when DM'ing a non-existing command: 
![image](https://user-images.githubusercontent.com/7468683/113781876-a2799200-9731-11eb-84b0-d6efd8718fb5.png)